ott/core/quad_problems.py

```diff
@@ -489,30 +478,27 @@ def update_linearization(
     Returns:
       Updated linear OT problem, a new local linearization of GW problem.
     """
-    rescale_factor = 1.0
     unbalanced_correction = 0.0
 
     marginal_1 = transport.marginal(axis=1)
     marginal_2 = transport.marginal(axis=0)
     marginal_cost = self.marginal_dependent_cost(marginal_1, marginal_2)
 
+    transport_matrix = transport.matrix
```

Comment:
maybe add a comment that this will likely break down at this step for LR approaches.

Reply:
Think there's no need for the comment, as LR approach will use `update_lr_linearization/update_lr_geometry`. But afaik in these functions don't handle the unbalanced correction.

Reply:
good point! indeed, this was not needed.